### ISSUE TITLE
Guidance on removing non-revisioned Istio Install

### ISSUE THREAD

**Issue by bdschaap** [nmnellis/istio-upgrade-demo#2]
Good article.  I wanted to see if you had any additional advice on removing the non-revisioned Istio deployment that owns the LoadBalancer service.  I realized that removing the non-revisioned control plane ends up removing that service which causes downtime even though a separate LoadBalancer service was defined outside of the new control plane that was deployed.

https://thenewstack.io/upgrading-istio-without-downtime/

"If your current Istio deployment owns the LoadBalanced service, you will have to take extra care when removing the existing infrastructure. It may be easier in some cases to migrate to a new LoadBalanced service."

**Comment by nmnellis** [comment]
To remove the old Istio deployment, you have to generate the yamls locally and parse through the ones you need to delete. Its a manual process because you dont want to delete the CRDs and that ingress LB most likely.

**Comment by bdschaap** [comment]
Yes I think my ingress LB was wiped out when I deleted non-revisioned istio-operator deployment as I was testing out that part of the process.  I'm not sure how to avoid that even with the guidance you suggest above.  I'm trying to avoid downtime for my customers and the only alternative I can think of to avoid downtime is to 1) provision a new LoadBalancer service with new ip's for ingress, 2) repoint DNS, 3) remove the non-revisioned istio-operator deployment which removes the old ingress LB.
